tests/sitemaps/test-class-wpseo-sitemaps-cache-data.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * @package WPSEO\Unittests
+ */
+
+/**
+ * Class WPSEO_Sitemaps_Test
+ */
+class WPSEO_Sitemaps_Cache_Data_Test extends WPSEO_UnitTestCase {
+
+	/**
+	 * @var WPSEO_Sitemap_Cache_Data subject to test against.
+	 */
+	private $subject;
+
+	/**
+	 * Create subject instance
+	 */
+	public function setUp() {
+		parent::setUp();
+		$this->subject = new WPSEO_Sitemap_Cache_Data();
+	}
+
+	/**
+	 * Test getting/setting sitemap
+	 *
+	 * @covers WPSEO_Sitemap_Cache_Data::set_sitemap()
+	 * @covers WPSEO_Sitemap_Cache_Data::get_sitemap()
+	 */
+	public function test_get_set_sitemap() {
+		$sitemap = 'this is a sitemap';
+
+		$this->subject->set_sitemap( $sitemap );
+		$this->assertEquals( $sitemap, $this->subject->get_sitemap() );
+	}
+
+	/**
+	 * Setting a sitemap that is not a string
+	 *
+	 * @covers WPSEO_Sitemap_Cache_Data::get_sitemap()
+	 * @covers WPSEO_Sitemap_Cache_Data::is_usable()
+	 */
+	public function test_set_sitemap_not_string() {
+		$sitemap      = new StdClass();
+		$sitemap->bla = 'hoi';
```

Comment:
Please use testing variables that are more descriptive. `$sitemap->bla = 'hoi';` doesn't make sense, especially if someone who's not Dutch, will be looking into this.